mailing_list/migrations/0014_add_new_permissions_for_roles.py

```diff
@@ -0,0 +1,91 @@
+from __future__ import unicode_literals
+import itertools
+
+from django.db import migrations
+
+# Adds new roles to the lti_school_permissions table and removes all isites migration permission records.
```

Comment:
Nipick: Fix comment to remove the reference to iSites migration as it maye be confusing later

Reply:
Good catch!